### DISCUSSION TITLE
Request for Urgent Assistance with GitHub Account Recovery

### DISCUSSION THREAD

**Discussion by kjpbrandnew** [community/community#144497] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear Support Team,

I am the owner of the GitHub account bigkjp97. Due to a job change, I have not logged into this account for several years. Recently, a friend informed me that my website, http://bigkjp97.github.io/, is now linking to an illegal betting site. I am extremely concerned, as I don't want my site to be used for fraudulent purposes.

Unfortunately, when attempting to regain access, I encountered a few obstacles:

1. The account requires two-factor authentication (2FA), but I no longer have access to my previous devices, so I cannot complete the 2FA process.

2. I accidentally unlinked my email from the account during this process, so I no longer have access through that method either.

I do still have the account name and password. I would be incredibly grateful if you could provide guidance on how to recover my account under these circumstances. This is an urgent matter, as I want to resolve this security issue as soon as possible.

Thank you very much for your help.

Best regards,
bigkjp97

**Comment by kjpbrandnew** [comment, marked as answer]
finally I find my github-recovery-codes.txt and delete my account.

**Reply by thannthann60-lab** [reply]
pattaya168.com

**Reply by thannthann60-lab** [reply]
pattaya168.com
